pkg/view/component/port.go

```diff
@@ -135,7 +135,7 @@ func describeButton(namespace, apiVersion, kind, name string, port int, pfs Port
 	}
 
 	buttonGroup.AddButton(
-		NewButton(buttonText, action.CreatePayload(actionName, payload)),
+		*NewButton(buttonText, action.CreatePayload(actionName, payload)),
```

Comment:
`NewButton` already returns a pointer to the button (`*Button` is the return type), you shouldn't need the `*`